### ISSUE TITLE
Publish v2.6

### ISSUE THREAD

**Issue by wenfangdu** [enyancc/vscode-ext-color-highlight#184]
@ferretwithaberet Dude, I saw you committed the v2.6 code but didn't publish it, have you got access to publish it?

If you do please publish it, if not do you have any fork version of this in the VSCode marketplace?

**Comment by wangyunduo** [comment]
Hi, there 👋  I notice that this repo has not had any new commits for the past year, and version 2.6.0, submitted last year, has not been released. Therefore, I forked the repo and developed a new extension based on version 2.6.0, named Color Highlight - Hue Show Better Version.

The extension [Color Highlight - Hue Show Better Version](https://marketplace.visualstudio.com/items?itemName=Yunduo.color-highlight-css-color-4) can solve your issue. Feel free to try it out : )
https://marketplace.visualstudio.com/items?itemName=Yunduo.color-highlight-css-color-4

**Comment by dzienisz** [comment]
these repo should be archived or they should find new maintainer

**Comment by ferretwithaberet** [comment]
> these repo should be archived or they should find new maintainer

Yeah, I am sorry. I was busy with college and could not maintain anymore. I was also planning to make a new extension based on this one (was planning to name it ColorFool, a play on words from Colorful) to resolve some issues, but I might just contribute to yours if I will have some spare time.
